web/webkit/src/main/scala/net/liftweb/http/CometActor.scala

```diff
@@ -683,7 +683,7 @@ trait BaseCometActor extends LiftActor with LiftCometActor with CssBindImplicits
    */
   def jsonToIncludeInCode: JsCmd = _jsonToIncludeCode
 
-  private lazy val (_sendJson, _jsonToIncludeCode) = S.createJsonFunc(Full(_defaultPrefix), onJsonError, receiveJson _)
+  private lazy val (_sendJson, _jsonToIncludeCode) = S.createJsonFunc(Full(_defaultPrefix), onJsonError, () => receiveJson)
```

Comment:
I hadn't noticed this yet in the work I've been doing on Scala upgrades (probably just missed it). To confirm my understanding, using `_` to notate that we're not applying a function is now deprecated?

Reply:
It is not globally, it's deprecated only for the case where the method doesn't have parens: 
- `def foo: Boolean = true` => need to use `() => foo`
- `def foo(): Boolean = true` => can still use `foo _`